saleor/graphql/webhook/mixins.py

```diff
@@ -0,0 +1,21 @@
+from django.core.exceptions import ValidationError
+
+from saleor.webhook.error_codes import WebhookErrorCode
+from saleor.webhook.event_types import WebhookEventAsyncType
+
+
+class NotifyUserEventValidationMixin:
+    @classmethod
+    def validate_events(cls, events):
+        # NOTIFY_USER needs to be the only one event registered per webhook.
+        # This solves issue temporarily. NOTIFY_USER will be deprecated in the future.
+        if WebhookEventAsyncType.NOTIFY_USER in events and len(events) > 1:
+            raise ValidationError(
+                {
+                    "async_events": ValidationError(
+                        "Notify User event has to be the only one event per webhook.",
+                        code=WebhookErrorCode.INVALID.value,
```

Comment:
I also think this is a very specific error, and providing a dedicated error code might be better. Then we could open a ticket in the dashboard to handle this specific error and provide a correct error message. Otherwise, the INVALID code may result in a too-generic error message, not clear to the user.

Reply:
do we have any convention for bug fix error codes? Or just something like `COMBINE_FORBBIDEN` is enough?

Reply:
You can see examples in `error_codes.py` files, some codes are simple and some are very specific. Here I would go with something descriptive like: `INVALID_NOTIFY_WITH_SUBSCRIPTION` or `CANNOT_COMBINE_NOTIFY_AND_SUBSCRIPTION`.